frontend/components/analysis/ListView.vue

```diff
@@ -0,0 +1,194 @@
+const tt = (s: string) => t(`${prefix}.${s}`)
+const expandedRows = useState<EditorObject[]>(`${prefix}.expandedRows`, () => [])
+const selectedRows = computed<EditorObject[]>({
+  get: () => {
+    const ids = selectedAnalysisIDs.value
+    return editorObjects.value.filter((editorObject) => ids.includes(editorObject.id))
+  },
+  set: (value: EditorObject[]) => {
+    const ids = value.map((row) => row.id)
+    ids.sort()
+    selectedAnalysisIDs.value = ids
+  },
+})
+
+const editorObjects = computed<EditorObject[]>(() => props.analyses.map((item) => ({ ...analysisEditor(item, i18n), id: item.id })))
+
+const selectedAnalyses = computed<Analysis[]>(() => selectedRows.value.map((row) => row.currentValue.value))
+
+const saveChanges = (id: string) => {
+  const index = editorObjects.value.findIndex((editorObject) => editorObject.id === id)
+  const eo = presentOrFileBug(editorObjects.value[index])
+  return withLoading(
+    () => pactaClient.updateAnalysis(id, eo.changes.value).then(refresh),
+    `${prefix}.saveChanges`,
+  )
+}
+
+const deleteAnalysis = (id: string) => withLoading(
+  () => pactaClient.deleteAnalysis(id),
+  `${prefix}.deleteAnalysis`,
+)
+const deleteSelected = () => Promise.all([selectedRows.value.map((row) => deleteAnalysis(row.id))]).then(refresh)
+</script>
+
+<template>
+  <div class="flex flex-column gap-3">
+    <div class="flex gap-2 flex-wrap">
+      <PVButton
+        icon="pi pi-refresh"
+        class="p-button-outlined p-button-secondary p-button-sm"
+        :label="tt('Refresh')"
+        @click="refresh"
+      />
+      <PVButton
+        :disabled="!selectedRows || selectedRows.length === 0"
+        icon="pi pi-trash"
+        class="p-button-outlined p-button-danger p-button-sm"
+        :label="tt('Delete') + selectedCountSuffix(selectedRows)"
+        @click="deleteSelected"
+      />
+    </div>
+    <PVDataTable
+      v-model:selection="selectedRows"
+      v-model:expanded-rows="expandedRows"
+      :value="editorObjects"
+      data-key="id"
+      size="small"
+      sort-field="editorValues.value.createdAt.originalValue"
+      :sort-order="-1"
+    >
+      <PVColumn selection-mode="multiple" />
+      <PVColumn
+        field="editorValues.value.createdAt.originalValue"
+        :header="tt('Created At')"
+        sortable
+      >
+        <template #body="slotProps">
+          {{ humanReadableTimeFromStandardString(slotProps.data.editorValues.value.createdAt.originalValue).value }}
+        </template>
+      </PVColumn>
+      <PVColumn
+        field="editorValues.value.name.originalValue"
+        sortable
+        :header="tt('Name')"
+      />
+      <PVColumn
+        :header="tt('View')"
+      >
+        <template #body="slotProps">
+          <LinkButton
+            icon="pi pi-external-link"
+            class="p-button-outlined p-button-xs"
+            :label="tt('View')"
+            :to="`${apiServerURL}/report/${slotProps.data.id}`"
```

Comment:
Discussed offline, add a slash here and make the non-slashed variety redirect to the slashed one, so that relative assets work

```suggestion
            :to="`${apiServerURL}/report/${slotProps.data.id}/`"
```

Reply:
Done!